index.html

```diff
@@ -17,158 +17,169 @@
         <a class="logo" href="#">Welcome</a>
         <nav>
             <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+            <ul class="menutabs">
+                <a href="#"><li>Portfolio</li></a>
+                <a href="#"><li>About</li></a>
+                <a href="#"><li>Portfolio</li></a>
+            </ul>
         </nav>
     </header>
     <!-- Headline Section-->
     <section class="headline">
-        <div class="container" id="welcometext">
-            <h1>Hey there. I'm Cristian</h1>
-            <h2 id="highlight">I'm a software developer</h2>
-        </div>
-        <div class="container">
-            <p>
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-                don’t hestiate to contact me.
-            </p>
-        </div>
-        <div class="container">
-            <ul>
-                <li class="socialmediaicons">
-                    <a href="https://twitter.com/"><img src="./icons/socialmedia/twitter.svg" alt="Twitter"></a>
-                </li>
-                <li class="socialmediaicons">
-                    <a href="https://www.linkedin.com/"><img src="./icons/socialmedia/linkedinicon.svg" alt="LinkedIn"></a>
-                </li>
-                <li class="socialmediaicons">
-                    <a href="https://medium.com/"><img src="./icons/socialmedia/medium.svg" alt="Medium"></a>
-                </li>
-                <li class="socialmediaicons">
-                    <a href="https://github.com/"><img src="./icons/socialmedia/github.svg" alt="GitHub"></a>
-                </li>
-                <li class="socialmediaicons">
-                    <a href="https://www.angellist.com/"><img src="./icons/socialmedia/angellist.svg" alt="Angel List"></a>
-                </li>
-            </ul>
-        </div>
-        <div class="container">
-            <a href="#nextSection"><img src="./icons/continue_button.png" alt="Continue"></a>
+        <div class="headlinecontent">
+            <div class="container" id="welcometext">
+                <h1>Hey there. I'm Cristian</h1>
+                <h2 id="highlight">I'm a software developer</h2>
+            </div>
+            <div class="container">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
```

Comment:
Also make sure that the text in your project scales with screen size you can use relative units like rem for font size instead of pixels.

**figma:**

![figma design desktop marked](https://user-images.githubusercontent.com/59707859/220775458-a29fc323-23dc-4afd-af4e-57686a9bdc66.png)

**your project**

![Student desktop headline marked](https://user-images.githubusercontent.com/59707859/220775521-33e335c0-5491-4c00-a9a0-06fa5e734d7c.png)